### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by sujithmn**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,57 +1,40 @@
...
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+# misc
+/.sass-cache
+/connect.lock
+/coverage
+/libpeerconnection.log
```
Comment (dario-piotrowicz) [inline .gitignore:29]: what's this? seems like something chrome generates in your home? (https://superuser.com/questions/627903/what-is-the-file-libpeerconnection-log)

why should it be included here? :confused:

File: .gitignore [hunk 2]
```diff
@@ -1,57 +1,40 @@
...
-
-# Ignore npm/yarn debug log
+*.sublime-workspace
+
+# IDE - VSCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```
Comment (dario-piotrowicz) [inline .gitignore:23]: why would we want to commit the vscode settings and stuff? :confused:

File: .gitignore [hunk 3]
```diff
@@ -1,57 +1,40 @@
-.DS_STORE
-
-/dist/
-/bazel-out
```
Comment (dario-piotrowicz) [inline .gitignore:4]: sorry, why is bazel-out removed?

**Comment by dario-piotrowicz** [comment]
Hi @sujithmn :slightly_smiling_face: 

Sorry I am not sure I am understanding your PR, why are you changing the gitignore file?
Could you please give a bit of context to these changes?

Your PR description does not seem to have an issue linked and in the description itself there is no explanation about why the changes are necessary :confused: 

Also I would really not consider the gitignore file as documentation (as you checked in the PR description)

(and you're commit message is not following the guidelines)

**Comment by petebacondarwin** [comment]
Closing as there is no explanation of why this change should land, and moreover the `.vscode` files should explicitly not be committed to the repository, since they can vary per developer. There are "recommended" files for that purpose.

**Pull request closed without merging**